AZ3166/src/libraries/AzureIoT/src/OTAUtils.cpp

```diff
@@ -0,0 +1,119 @@
+#include "OTAUtils.h"
+#include "parson.h"
+#include "DevKitMQTTClient.h"
+#include "ctype.h"
+static char *currentFirmwareVersion = NULL;
+static FW_INFO* otaFwInfo = NULL;
```

Comment:
Why need pointer here? Consider FW_INFO otaFwInfo directly and then check if member variable is NULL.